ckanext/data_qld/templates/package/snippets/package_form.html

```diff
@@ -1,21 +1,70 @@
-{% ckan_extends %}
-
-
-{% block delete_button %}
-  {% if h.check_access('package_delete', {'id': data.id}) and not data.state == 'deleted' and not h.data_qld_data_driven_application(data.data_driven_application)  %}
-    <a id='dataset-delete-button' disabled="disabled"
-        class="btn btn-danger pull-left"
-        {% if h.is_ckan_29() %}
-          href="{{ h.url_for('dataset.delete', id=data.id) }}"
-        {% else %}
-          href="{% url_for controller='package', action='delete', id=data.id %}"
-        {% endif %}
-        data-module="confirm-action"
-        data-module-content="{{ _('Deleting this dataset will prevent future use of the dataset’s name. It is highly recommended that dataset issues are rectified rather than deleting and creating a new dataset with the same information.') }}"
-        data-module-template="{{ h.get_deletion_reason_template() }}">
-        {% block delete_button_text %}{{ _('Delete') }}{% endblock %}
-    </a>
-    {% set type = 'asset' if h.is_ckan_29() else 'resource' %}
-    {% include "package/snippets/dataset_deletion_" ~ type ~ ".html" %}
-  {% endif %}
-{% endblock %}
+{% import 'macros/form.html' as form %}
```

Comment:
Why do we need to override the entire template instead of extending it? This will mean more work in the future to keep our template synchronised with new versions of CKAN.

Reply:
I had to do it, to change form enctype. I'll appreciate advice how to do it properly.

Reply:
Okay. As Will mentioned, we could adjust our CKAN fork (https://github.com/qld-gov-au/ckan/) to put the encoding in a block so it can be overridden, which would be cleaner.

Reply:
@ThrawnCA , I see that you are using some tag system here, e.g. `ckan-2.9.5-qgov.9`. Do I need to create `ckan-2.9.5-qgov.10` with a tweaked `package_form.html`?

Reply:
The version is set here: https://github.com/qld-gov-au/ckanext-data-qld/blob/develop/.circleci/build.sh#L19

Reply:
Note that any development on the CKAN fork should be done on the https://github.com/qld-gov-au/ckan/tree/qgov-master-2.9.5 branch.